### DISCUSSION TITLE
Can I replace next-routes with the new versions?

### DISCUSSION THREAD

**Discussion by nihiluis** [vercel/next.js#13368] [Help] [answered]
I was using next 8 when I installed next-routes to handle these routes. Can I replace it with native next.js nowadays?

```js
const Routes = require("next-routes")

const routes = new Routes()
  .add("invite", "/invite/:inviteId")
  .add("team", "/team/:teamId")
  .add("review/master", "/review/master/:reviewMasterId")
  .add("user", "/user/:userId")
  .add("login")
  .add("leaders")

module.exports = routes
```

**Comment by nihiluis** [comment, marked as answer]
As I see it's supported by this? https://nextjs.org/docs/routing/dynamic-routes

**Reply by Timer** [reply]
Yes, the new dynamic routes feature is what you're looking for!

**Reply by nihiluis** [reply]
Thank you papi

**Comment by revskill10** [comment]
Yes with `rewrites` feature you can.

**Reply by Timer** [reply]
Dynamic routes should be preferred over rewrites. Rewrites is a last-resort for things not modelable in the file system.
